Add meta description to some pages (#4534)
wazuh/wazuh-documentation: source/docker/upgrade-guide.rst

.. Copyright (C) 2021 Wazuh, Inc.

.. meta::

>   :description: Docker, an open source project that automates the deployment of different applications inside software containers. Learn more about it here. 

Reviewer: Sentence changed by:

.. meta::
  :description: With Docker, you can automate the deployment of applications inside containers. Learn about the upgrade of a cluster running version previous to 4.0 here.

>   :description: Docker is an open source project that automates the deployment of different applications inside software containers. Learn more about the upgrade guide here. 

Reviewer: ```suggestion
  :description: Docker, an open source project that automates the deployment of different applications inside software containers. Learn more about it here. 
```
